examples/inverter_example.py

```diff
@@ -1,17 +1,37 @@
 """ Example code to run the forecast"""
+import pandas as pd
+from datetime import datetime
 from quartz_solar_forecast.forecast import run_forecast
 from quartz_solar_forecast.pydantic_models import PVSite
-from datetime import datetime, timedelta
+
+# Set plotly backend to be plotly, you might have to install plotly
+pd.options.plotting.backend = "plotly"
+
 
 def main():
-    # make input data
-    site = PVSite(latitude=51.75, longitude=-1.25, capacity_kwp=1.25, inverter_type="enphase")
-    
-    ts = datetime.today() - timedelta(weeks=1)
-    predictions_df = run_forecast(site=site, ts=ts, nwp_source="icon")
-
-    print(predictions_df)
-    print(f"Max: {predictions_df['power_wh'].max()}")
+
+    ts = ts = pd.to_datetime(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
```

Comment:
I think there is a typo here got two `ts`s

Reply:
> I think there is a typo here got two `ts`s assigned

My bad, fixed this in my latest commit